system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -169,6 +169,72 @@
                    :Term "ENVIRONMENTAL IMPACTS"
                    :Topic "HUMAN DIMENSIONS"}]))))))
 
+(deftest bulk-update-science-keywords-2
+  ;; This test is the same as the previous bulk-update-science-keywords test except
+  ;; that it shows that update-value could be an array of objects, as well as object, when update-type is "ADD_TO_EXISTING" 
```

Comment:
A few of the lines are more than 100 chars so for example we do not see the full line without scrolling in the Github review.

Reply:
Done.